java/kotlin-extractor/kotlin_plugin_versions.py

```diff
@@ -26,11 +27,11 @@ class KotlincNotFoundException(Exception):
     pass
 
 def get_single_version(fakeVersionOutput = None):
-    # TODO: `shell=True` is a workaround to get CI working on Windows. It breaks the build on Linux.
-    try:
-        versionOutput = subprocess.run(['kotlinc', '-version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, shell=is_windows()).stderr if fakeVersionOutput is None else fakeVersionOutput
-    except FileNotFoundError as e:
-        raise KotlincNotFoundException(e)
+    # kotlinc might be kotlinc.bat or kotlinc.cmd on Windows, so we use `which` to find out what it is
+    kotlinc = shutil.which('kotlinc')
+    if kotlinc is None:
+        raise KotlincNotFoundException()
+    versionOutput = subprocess.run(['kotlinc', '-version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True).stderr if fakeVersionOutput is None else fakeVersionOutput
```

Comment:
Is this `'kotlinc'` correct? Shouldn't we use the path in the `kotlinc` variable instead?

Reply:
ta, fixed